diff --git a/pyproject.toml b/pyproject.toml
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -9,7 +9,11 @@
 requires-python = ">=3.9"
 license = {text = "MIT"}
 dynamic = ["version"]
-dependencies = ["typing_extensions", "websockets"]
+dependencies = [
+  "typing_extensions",
+  "websockets",
+  "opentrons-shared-data==9.1.0+unilabos.np2.1",
+]
 
 [project.optional-dependencies]
 serial = ["pyserial"]
